pkg/runner/runner.go

```diff
@@ -169,7 +169,21 @@ func RunBundle(action string, ns string, bundleName string, sandboxRole string,
 	}
 
 	// TODO: return nil
-	return
+	return pn, nil
```

Comment:
Ah, now I see it. I think it would be helpful if we named the return value since the function name doesn't explicitly tell us what kind of return value to expect in the string.